### PR TITLE
Updating meshery-ci meeting day

### PR DESCRIPTION
**Pull request by Saim-Safdar**

### CODE REVIEW

**Comment by leecalcote** [comment]
> @Saim-Safdar DCO check is failing, it would be great if you could please follow these steps to resolve this issue:-
> https://github.com/layer5io/meshery.io/pull/163/checks?check_run_id=616677141

In addition, as a permanent fix, you might take advantage of these instructions - https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin

**Comment by leecalcote** [comment]
@rav121... fyi

File: _includes/partials/connect.html [hunk 1]
```diff
@@ -6,11 +6,11 @@
         
         <p class="community-block">
             <ul class="project-block">
-                <li><em>Join</em> <a href="https://docs.google.com/document/d/1c07UO9dS7_tFD-ClCWHIrEzRnzUJoFQ10EzfJTpS7FY/edit?usp=sharing">weekly community meeting</a> on <a href="/assets/projects/meshery/Meshery-Community-Meeting.ics">Fridays from 10am to 11am Central</a>.</li>
+                <li><em>Join</em> <a href="https://docs.google.com/document/d/1GrVdGHZAYeu6wHNLLoiaKNqBtk7enXE9XeDRCvdA4bY/edit?usp=sharing">meshery-ci weekly community meeting</a> on <a href="/assets/projects/meshery/Meshery-Community-Meeting.ics">Mondays from 10am to 11am Central</a>.</li>
```
Comment (kushthedude) [inline _includes/partials/connect.html:9]: @Saim-Safdar I would suggest adding a new meeting, rather than renaming community meeting to CI meeting.

Comment (Saim-Safdar) [reply _includes/partials/connect.html:9]: @kushthedude Yes great suggestion I'll do that.

**Comment by Saim-Safdar** [comment]
@leecalcote I think its better to automate things which we do repeatedly. So far when we create a working group we have to create a new channel in slack and next step might be to update an html and reference docs. We have to choose certain criteria when adding a new channel in slack when this criteria is met we can trigger auto generated html with updated value and direct them to places where we want it to be. But first we have to identify the step before automate this thing. I think this will be the better approach moving farward.

**Comment by leecalcote** [comment]
@Saim-Safdar will you have another look here and ensure that the CI meeting details are called out in addition to the Community meeting details?

**Comment by leecalcote** [comment]
Fixed by #192

**Pull request closed without merging**